### DISCUSSION TITLE
classes/class_pckpacker

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#115] [User-contributed notes]
# classes/class_pckpacker

Inherits: RefCounted< Object Creates packages that can be loaded into a running project. Description: The PCKPacker is used to create packages that can be loaded into a running project using Projec...

https://docs.godotengine.org/en/stable/classes/class_pckpacker.html

**Comment by Repflez** [comment]
Specifying a key for an encrypted PCK is basically useless in practice as there's no current (as of 4.3) method to load an encrypted PCK with a different key from the one that was built in the export templates [until this PR gets merged](https://github.com/godotengine/godot/pull/91884).
